app/src/main/java/com/example/alex/pollsui/data/source/local/PollsFakeDataSource.kt

```diff
@@ -0,0 +1,75 @@
+package com.example.alex.pollsui.data.source.local
+
+import android.util.Log
+import com.example.alex.pollsui.data.Answer
+import com.example.alex.pollsui.data.Poll
+import com.example.alex.pollsui.data.Question
+import com.example.alex.pollsui.data.User
+import com.example.alex.pollsui.data.source.PollsDataSource
+
+class PollsFakeDataSource : PollsDataSource {
+
+    private val suffixes = mapOf(1 to "First", 2 to "Second", 3 to "Third", 4 to "Fourth")
+
+    private val polls by lazy {
+        Log.d("FakeData", "Init polls")
+        val res = LinkedHashMap<String, Poll>()
+        suffixes.keys.forEach {
+            val poll = generatePoll(it)
+            res[poll.id] = poll
+        }
+        res
+    }
+
+    private val questions by lazy {
+        Log.d("FakeData", "Init questions")
+        val res = ArrayList<Question>()
+        suffixes.keys.forEach {
+            val question = Question("Question".addSuffix(it))
+            question.answers.addAll(answers)
+            res.add(question)
+        }
+        res
+    }
+
+    private val answers by lazy {
+        Log.d("FakeData", "Init answers")
+        val res = ArrayList<Answer>()
+        suffixes.keys.forEach { res.add(Answer("Answer".addSuffix(it))) }
```

Comment:
I think it would be better to move forEach body to new line to keep it consistent